sensor/kubernetes/certrefresh/localscanner/certificate_requester.go

```diff
@@ -107,11 +111,30 @@ func (r *certificateRequesterImpl) send(ctx context.Context, requestID string) e
 	}
 }
 
-func receive(ctx context.Context, receiveC <-chan *central.IssueLocalScannerCertsResponse) (*central.IssueLocalScannerCertsResponse, error) {
+func convertToIssueCertsResponse(response *central.IssueLocalScannerCertsResponse) *certrequester.IssueCertsResponse {
```

Comment:
Hmm, this convert function works with both types that are external to this pkg. Wouldn't that fit better to one of the pkgs involved in the conversion? It looks like the best place for it would be `certrequester` (maybe even as a member function `NewFromResponse...`), but please tell me if I am wrong.

Reply:
I took your suggestion and moved the function there.

I also renamed `certrequester` to `certificates`, which is close to what you were suggesting. Somehow the plural sounds better to me, because we're always issuing a set of certs. And then I renamed the types of the package to avoid repetition.